modules/ROOT/pages/provision-adapter-configure-service-mesh-CRD.adoc

```diff
@@ -80,6 +80,61 @@ spec:
 . Verify the <<adapter_status, adapter status,>> using the following command:
 `asmctl adapter list`
 
+== Reference Kubernetes Secret
+
+Anypoint Service Mesh supports the use of Kubernetes secrets to store and manage sensitive information, such as your user credentials.
+
+*Task Prerequisites*
+
+You must have the required permission to create the Kubernetes secret.
```

Comment:
SUGGESTION:
Anypoint Service Mesh supports the use of Kubernetes secrets to store and manage sensitive information, such as your user credentials. You must have the required permission to create the Kubernetes secret.